main.js

```diff
@@ -0,0 +1,66 @@
+var PlayGame = {
+  
+  board:  [[5, 4, 3, 2, 1], [], []],
+  results: function () {
+    for (let i=0; i < this.board.length; i++) {
+      const rowResults = this.board[i].map(num => num.toString()).join(" ");
+      console.log ("---" + " "  + rowResults);
+    };
+  },
+  
+  checkWinner: function () {
+    const winningPeg = [5, 4, 3, 2, 1];
+    for (let i = 1; i < this.board.length; i++) {
+      const checkDiscsRow = this.board[i];
+      let flag = true;
+      checkDiscsRow.forEach((disc, index) => {
+        if(disc != winningPeg[index]) {
+          flag = false;
+        }
+      })
+        if (flag === true) {
```

Comment:
this should just be if (flag) because a condition evaluates to a boolean but since it is already in that form, you can just check the variable and not have to do a comparison